caddy.go

```diff
@@ -261,13 +263,28 @@ func changeConfig(method, path string, input []byte, ifMatchHeader string, force
 	return nil
 }
 
+func rollbackRawCfg() error {
+	var oldCfg any
+	err := json.Unmarshal(rawCfgJSON, &oldCfg)
+	rawCfg[rawConfigKey] = oldCfg
```

Comment:
obs: Potential nil assignment to `rawCfg`. Check for error before assigning to the map